### ISSUE TITLE
Discussion: drop of develop branch

### ISSUE THREAD

**Issue by rickmak** [likecoin/likecoin-chain#53]
Currently, I am very confuse on the master vs develop branch. It looks no specific development workflow.

If no objection, I will delete the develop branch and all change target for the next release should send PR to master.

For feature development, let keep current convention of prefix with `feat` which is clear.

For patch we need to release to existing chain, we should patch the current master too. So of the nature of the blockchain, we will only keep one production branch, which is codename, like current `sheungwan`, `fotan`. 

@williamchong @nnkken thought?

**Comment by rickmak** [comment]
Current default is master branch now. Will delete the develop branch.

**Comment by williamchong** [comment]
yes, for the `codename` branch should we use sementic prefix like `v1.x.x-fotan` for easier searching?

**Comment by rickmak** [comment]
I think major version tracking branch should be v1-fotan, v2-laichikok, v3-shatin, v4-taipo, etc?

**Comment by williamchong** [comment]
Yes that make sense

**Comment by ckxpress** [comment]
@nnkken and teammates had some previous discussions on the major version names.
p.s. i don't have any opinions on this.
